makefile

```diff
@@ -29,6 +29,14 @@ ifndef INSTALL_DIR
 	INSTALL_DIR=/usr
 endif
 
+ifndef INCLUDE_PATH
+	INCLUDE_PATH=${INSTALL_DIR}/include
+endif
+
+ifndef LIBRARY_PATH
+	LIBRARY_PATH=${INSTALL_DIR}/lib
+endif
+
```

Comment:
This is to let the user to install in other directories other than the default ones (/usr or /usr/local, I don't remember right now).